### DISCUSSION TITLE
auto create pq's via project-gc on firefox

### DISCUSSION THREAD

**Discussion by novw** [2Abendsegler/GClh#2179] [Questions] [answered]
The popups no longer happen on firefox 104.0.1, so creating pq's with the help of project gc's pq splitter no longer works.

I disabled all add-ons except tampermonkey, so its (probably) not any of the "other" addons.

**Comment by 2Abendsegler** [comment, marked as answer]
Thanks for your message. We have already an issue for this bug.  #2177
